src/services/sports/index.jsx

```
@@ -0,0 +1,400 @@
+import ProfileCard from "../../components/card/ProfileCard";
+import { copyLink } from "../../utils/copyLink";
+import {
+  addCommentOnSportsAndSports,
+  addSportsAndSports,
+  getSportsAndSportsComments,
+  getSportsAndSportsLikes,
+  getSportsAndSportsSubCategoryByCategory,
+  likeUnlikeSportsAndSports,
+} from "../../app/features/sportsandsports";
+
+export const AddSports = ({ setAddModal, dispatch, setReload, categoryId }) => {
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)